packages/core/lib/Actor.ts

```diff
@@ -222,5 +222,5 @@ export interface IActorTest {
  * Data interface for the type of an actor run result.
  */
 export interface IActorOutput {
-
+  context?: ActionContext;
```

Comment:
Instead of adding this here, could we add it to the output interface of the optimization bus?
As long as it's an optional field, this should still be backwards-compatible.

Reply:
Done!